backend/tcs-service/Repos/ReportsRepo.cs

```diff
@@ -42,16 +43,16 @@ public async Task<List<WeeklyVisitsViewModel>> WeeklyVisits(DateTime startWeek,
         public async Task<List<PeakHoursViewModel>> PeakHours(DateTime startWeek, DateTime endWeek)
              => await _db.Sessions.Where(x => x.InTime >= startWeek && x.InTime <= endWeek)
                 .GroupBy(x => x.InTime.Hour)
-                .Where(x => x.Count() > 1)
+                .Where(x => x.Count() >= 1)
                 .Select(x => new PeakHoursViewModel(x.Key, x.Count()))
                 .ToListAsync();
 
 
         public async Task<List<ClassTourReportViewModel>> ClassTours(DateTime startWeek, DateTime endWeek)
```

Comment:
We want to switch away from all the ViewModel stuff. Every view model used in ReportsRepo should be switch to DTOs.